dist/gaia-header.js

```diff
@@ -71,7 +71,8 @@ module.exports = function(config) {
 
   return {
     textWidth: textWidth,
-    fontSize: fontSize
+    fontSize: fontSize,
+    overflowing: textWidth > space
```

Comment:
I'll bump the font-fit version once https://github.com/gaia-components/font-fit/pull/1 is reviewed